delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/Storages.scala

```diff
@@ -751,4 +751,17 @@ object Storages {
     }
   }
 
+  /**
+    * Create an instance of [[EventExchange]] for [[StorageEvent]].
+    * @param storages storages operation bundle
+    */
+  def eventExchange(storages: Storages)(implicit crypto: Crypto, resolution: RemoteContextResolution): EventExchange =
+    EventExchange.create(
+      (event: StorageEvent) => storages.fetch(IriSegment(event.id), event.project).leftWiden[StorageRejection],
+      (event: StorageEvent, tag: TagLabel) =>
+        storages.fetchBy(IriSegment(event.id), event.project, tag).leftWiden[StorageRejection],
+      (storage: Storage) => storage.toExpandedJsonLd,
+      (storage: Storage) => Task.fromEither(Storage.encryptSource(storage.source, crypto)).hideErrors
```

Comment:
throwing in the internal error channel will stop any indexing stream unless there's specific handling for that... do you have any thoughts on this?

Reply:
This is fine. When this happens it means we messed up with some configuration. We want things to fail badly so that we realized something is wrong.

That said, it would be good to add some log (but I guess the stream should log failures already...so that should be fine)